wgpu-types/src/lib.rs

```diff
@@ -1177,116 +1179,123 @@ impl TextureFormat {
         let Linear = false;
         let Srgb = true;
 
-        let (features, sample_type, srgb, block_dimensions, block_size) = match self {
+        // Flags
+        let Basic = TextureUsage::COPY_SRC | TextureUsage::COPY_DST | TextureUsage::SAMPLED;
```

Comment:
why doing the weird naming scheme? locals should start with lower case

Reply:
This had justification at one point, but I don't remember it, and either way on rethink I don't agree with it, will change to conventional names.